src/modules/Workspaces/WorkspacesEditor/Utils/StringUtils.cs

```diff
@@ -10,7 +10,7 @@ public static class StringUtils
     {
         public static string UpperCamelCaseToDashCase(this string str)
         {
-            // If it's single letter variable, leave it as it is
+            // If it's a single letter variable, leave it as it is
```

Comment:
I think this was from looking over `it's` in general in search of a fancier rule. (The rule against `its'` is simple, but trying to formulate one for `it's ...` is harder.)